commit 23221973979cd8d85953ac9cafa9e38e95d00c3d (refs/changes/56/2956/1)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-11-16T11:05:14-08:00 (1 year, 11 months ago)
    
    ZAPI-812 Unable to provision kvm with vmapi version master-20171110T184239Z-ga60a380

diff --git a/lib/workflows/provision.js b/lib/workflows/provision.js
index 0cd988e..7bd6c05 100644
--- a/lib/workflows/provision.js
+++ b/lib/workflows/provision.js
@@ -372,7 +372,7 @@ function ensureImage(job, cb) {
     var ensurePayload = {};
 
     if (job.params['brand'] === 'kvm') {
-        ensurePayload.image_uuid = job.params.payload.disks[0].image_uuid;
+        ensurePayload.image_uuid = job.params.disks[0].image_uuid;
     } else {
         ensurePayload.image_uuid = job.params.image_uuid;
     }
